drivers/hmis_external_apis/public/schemas/form_definition.json

```diff
@@ -94,107 +219,600 @@
+        },
+        {
+          "if": {
+            "required": [
+              "type"
+            ],
+            "properties": {
+              "type": {
+                "const": "TIME_OF_DAY"
+              }
+            }
+          },
+          "then": {
+            "$ref": "#/$defs/genericInputFormNode"
+          }
+        }
+      ],
+      "unevaluatedProperties": false
+    },
+    "choiceFormNode": {
+      "allOf": [
+        {
+          "$ref": "#/$defs/genericInputFormNode"
+        },
+        {
+          "type": "object",
+          "oneOf": [
+            {
+              "required": [
+                "pick_list_reference"
+              ]
+            },
+            {
+              "required": [
+                "pick_list_options"
+              ]
+            }
+          ],
+          "properties": {
+            "pick_list_reference": {
+              "type": "string"
+            },
+            "pick_list_options": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "$ref": "#/$defs/pickListOption"
+              }
+            },
+            "component": {
+              "enum": [
+                "CHECKBOX",
+                "DROPDOWN",
+                "RADIO_BUTTONS",
+                "RADIO_BUTTONS_VERTICAL"
+              ]
+            }
+          }
+        }
+      ]
+    },
+    "booleanFormNode": {
+      "allOf": [
+        {
+          "$ref": "#/$defs/genericInputFormNode"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "component": {
+              "enum": [
+                "CHECKBOX"
+              ]
+            }
+          }
+        }
+      ]
+    },
+    "displayFormNode": {
+      "allOf": [
+        {
+          "$ref": "#/$defs/baseFormNode"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "autofill_values": {
+              "$comment": "used for display that includes calculations",
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "$ref": "#/$defs/autofillValues"
+              }
+            },
+            "read_only": {
+              "$comment": "TODO: unclear what this means in a GROUP, add docs",
+              "type": "boolean"
+            },
+            "component": {
+              "enum": [
+                "ALERT_ERROR",
+                "ALERT_SUCCESS",
+                "ALERT_INFO",
+                "ALERT_WARNING"
+              ]
+            },
+            "initial": {
+              "$comment": "used for display that includes calculations",
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "$ref": "#/$defs/initial"
+              }
+            },
+            "mapping": {
+              "$comment": "used for display that includes calculations",
+              "$ref": "#/$defs/mapping"
+            },
```

Comment:
We support this, but have a backlog issue to consider removing support for it: https://github.com/open-path/Green-River/issues/5964